src/pages/Product/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import ProductIcon from './ProductIcon/ProductIcon';
+import ProductDescription from './ProductDescription/ProductDescription';
+
+import { AiOutlinePlus } from 'react-icons/ai';
+
+import './ProductCard.scss';
+
+export class ProductCard extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  render() {
+    const { addCart, backgruonColor, handleCartButton } = this.props;
+
+    const { efficacy, name, icon, pillImage, description, quantity, price } =
+      this.props.productCard;
+
+    return (
+      <li className="ProductCard" style={{ backgroundColor: backgruonColor }}>
+        <Link to="/detail">
+          <header className="productCardHeader">
+            <div className="nameBox">
+              <h2>
+                {efficacy}
+                <br />
+                <strong>{name}</strong>
+              </h2>
+
+              <ul className="icon">
+                {icon &&
+                  icon.map(iconItem => {
+                    return (
+                      <ProductIcon
+                        key={iconItem.id}
+                        src={iconItem.src}
+                        alt={iconItem.alt}
+                      />
+                    );
+                  })}
```

Comment:
```suggestion
                  icon.map(({id,src,alt})=> {
                    return (
                      <ProductIcon
                        key={id}
                        src={src}
                        alt={alt}
                      />
                    );
                  })}
```
인자로 객체가 들어올 경우 바로 구조분해할당 할 수 있습니다.

Reply:
신비한 구조분해할당의 세계...! 저에게도 익숙해지는 시간이 얼른 오길🙏